service-deployment/bootstrap/prometheus/start.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+pushd $(dirname "$0") > /dev/null
+
+NAMESPACE=${NAMESPACE:-monitor}
+KUBECTL="kubectl  --namespace=\"${NAMESPACE}\""
+eval "kubectl create namespace \"${NAMESPACE}\""
+
+eval "${KUBECTL} create -f node-exporter-ds.yaml"
+eval "${KUBECTL} get pods"
```

Comment:
don't need this. if you really want to make sure it is started. please refer to [https://github.com/Microsoft/pai/blob/master/service-deployment/kubernetesTool/servicestatus.py](https://github.com/Microsoft/pai/blob/master/service-deployment/kubernetesTool/servicestatus.py)